src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = (props) => {
+  const {
+    person,
+  } = props;
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age
+        ? (<p className="Person__age">{`I am ${age}`}</p>)
+        : null
+      }
+
+      {isMarried
+        ? (<p className="Person__partner">{`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`}</p>)
+        : (<p className="Person__partner">I am not married</p>)
+      }
+    </section>
```

Comment:
Good, but for better understanding use a constant with {sex === 'm' ? 'wife' : 'husband'}